tests/functional/local_endpoint/test_server.py

```diff
@@ -0,0 +1,78 @@
+from globus_sdk import LocalGlobusConnectServer
+
+
+def test_info_dict_from_nonexistent_file_is_none(tmp_path):
+    info_path = tmp_path / "info.json"
+    gcs = LocalGlobusConnectServer(info_path=info_path)
+    assert gcs.info_dict is None
+
+
+def test_info_dict_from_non_json_file_is_none(tmp_path):
+    info_path = tmp_path / "info.json"
+    info_path.write_text("{")
```

Comment:
A failure case for invalid Unicode is needed here as well. Perhaps write a binary file with:

```python
bytes.fromhex("1BAD DEC0DE")
```

Eh? EH?

Reply:
I cannot turn down `1BAD DEC0DE`. I don't have that kind of willpower. Expect to see it later today, when I can circle back on this.